restores autocomlete options for arguments

LeStarch opened the pull request:
| | |
|:---|:---|
|**_Originating Project/Creator_**| |
|**_Affected Component_**|  |
|**_Affected Architectures(s)_**|  |
|**_Related Issue(s)_**|  |
|**_Has Unit Tests (y/n)_**|  |
|**_Builds Without Errors (y/n)_**|  |
|**_Unit Tests Pass (y/n)_**|  |
|**_Documentation Included (y/n)_**|  |

---
## Change Description

Fixes nasa/fprime#1689.

Changes in the data module broke autocomplete for arguments.

LeStarch committed:
restores autocomlete options for arguments

thomas-bc reviewed (dismissed):
Looks good!

Just as a note, I believe there are two other remnants of the legacy data format. They don't cause hardcore bugs, but the validation of an ENUM input is not happening because of it in the Command tab.

https://github.com/fprime-community/fprime-gds/blob/a744b42b3caa97db1ec406c13407fabfd9a8ba65/src/fprime_gds/flask/static/js/validate.js#L71-L72

https://github.com/fprime-community/fprime-gds/blob/a744b42b3caa97db1ec406c13407fabfd9a8ba65/src/fprime_gds/flask/static/js/vue-support/command.js#L177

LeStarch committed:
fixing misc validation as requested

LeStarch commented:
@thomas-bc I fixed both of the incorrect usages of "possible".  I also fixed an unrelated validation bug with floats that I noticed in the same code block.

thomas-bc reviewed (approved):
Awesome! LGTM
